### ISSUE TITLE
Selecting words

### ISSUE THREAD

**Issue by pixla** [Nitrino/easysubs#161]
Hello sir, I'm enjoying your extension ! Please could you add possibility select 2 words. For example German verbs have two part in sentence. Will be your add-on support https://www.disneyplus.com/ ?

Thank you very much, You do very good job

**Comment by tim2371** [comment]
Yes, that would be nice. It's even better to be able to select more than 2 words.
